js/adapt-contrib-trickle.js

```diff
@@ -1,8 +1,6 @@
 define([
     'coreJS/adapt',
```

Comment:
May as well switch to `'core/js/adapt'` while we're here.

Reply:
and in adaptModelExtension.js too